diff --git a/src/systems/physics-system.js b/src/systems/physics-system.js
index aa9e2a99..6ad3cd2f 100644
--- a/src/systems/physics-system.js
+++ b/src/systems/physics-system.js
@@ -41,7 +41,7 @@ export class PhysicsSystem {
         type: MESSAGE_TYPES.INIT,
         worldConfig: WORLD_CONFIG,
         arrayBuffer,
-        wasmUrl: new URL(ammoWasmUrl, configs.BASE_ASSETS_PATH || window.location).href
+        wasmUrl: new URL(ammoWasmUrl, window.location).href
       },
       [arrayBuffer]
     );
