diff --git a/src/utils/phoenix-utils.js b/src/utils/phoenix-utils.js
index d3b96e74..f17a0db3 100644
--- a/src/utils/phoenix-utils.js
+++ b/src/utils/phoenix-utils.js
@@ -187,7 +187,7 @@ export async function createAndRedirectToNewHub(name, sceneId, replace) {
   }
 
   if (isLocalClient()) {
-    url = `/hub.html?hub_id=${hub.hub_id}`;
+    url = process.env.BASE_ASSETS_PATH + `hub.html?hub_id=${hub.hub_id}`;
   }
 
   if (replace) {
diff --git a/webpack.config.js b/webpack.config.js
index e6361977..48090019 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -615,4 +615,5 @@ module.exports = async (env, argv) => {
           POSTGREST_SERVER: process.env.POSTGREST_SERVER,
+          BASE_ASSETS_PATH: process.env.BASE_ASSETS_PATH,
           APP_CONFIG: appConfig
         })
       })
