diff --git a/src/index.js b/src/index.js
index 423afb79..0a10ed53 100644
--- a/src/index.js
+++ b/src/index.js
@@ -14,6 +14,11 @@ import jwtDecode from "jwt-decode";
 import "./utils/theme";
 import configs from "./utils/configs";
 
+if (window.location.protocol != "https:") {
+  window.location.protocol = "https:";
+  
+}
+
 const qs = new URLSearchParams(location.search);
 registerTelemetry("/home", "Hubs Home Page");
 
