diff --git a/src/utils/media-url-utils.js b/src/utils/media-url-utils.js
index 669537f0..02c6f2c7 100644
--- a/src/utils/media-url-utils.js
+++ b/src/utils/media-url-utils.js
@@ -5,6 +5,7 @@ const nonCorsProxyDomains = (configs.NON_CORS_PROXY_DOMAINS || "").split(",");
 if (configs.CORS_PROXY_SERVER) {
   nonCorsProxyDomains.push(configs.CORS_PROXY_SERVER);
 }
+nonCorsProxyDomains.push(document.location.hostname)
 
 const commonKnownContentTypes = {
   gltf: "model/gltf",
